MediaPicker/src/main/java/org/wordpress/android/util/WPMediaUtilsWrapper.kt

```diff
@@ -0,0 +1,14 @@
+package org.wordpress.android.util
+
+import android.content.Context
+import android.net.Uri
+
+class WPMediaUtilsWrapper(private val context: Context) {
```

Comment:
Related issue: [Make WPMediaUtils and WPMediaUtilsWrapper consistent (#28)](https://github.com/wordpress-mobile/WordPress-MediaPicker-Android/issues/28)